anesthetic/samples.py

```diff
@@ -1334,6 +1372,168 @@ def recompute(self, logL_birth=None, inplace=False):
         else:
             return samples.__finalize__(self, "recompute")
 
+    def plot_types(self):
+        """
+        Get types of plots supported by this class.
+
+        Returns
+        -------
+        tuple[str]
+        """
+        return 'live', 'posterior'
+
+    def points_to_plot(self, plot_type, label, evolution, beta, base_color):
+        """
+        Get samples for plotting.
+
+        Parameters
+        ----------
+        plot_type: str
+            see plot_types() for supported types.
+        label: str
+            column to plot
+        evolution: int
+            iteration to plot
+        beta: float
+            temperature
+        base_color:
+            base_color used to create color palette
+
+        Returns
+        -------
+        List[array-like]: list of points to plot
+        List[tuple[float]: colors to use
+        """
+        if plot_type == 'posterior':
+            return [self.posterior_points(beta)[label]], [base_color]
+        elif plot_type == 'live':
+            logL = self.logL.iloc[evolution]
+            return [self.live_points(logL)[label]], [base_color]
+        else:
+            raise ValueError("plot_type not supported")
```

Comment:
This, too, does not feel like it should be a method of `NestedSamples`. Too GUI specific.